src/execution/execute.ts

```diff
@@ -826,32 +825,73 @@ function completeAbstractValue(
 
   return completeObjectValue(
     exeContext,
-    ensureValidRuntimeType(
-      runtimeType,
-      exeContext,
-      returnType,
-      fieldNodes,
-      info,
-      result,
-    ),
+    runtimeType,
     fieldNodes,
     info,
     path,
     result,
   );
 }
 
+function resolveType(
```

Comment:
I would suggest making it one argument that is an object

Reply:
Hmmm, I think most of the code in this file does not have named arguments, see for example nearby `ensureValidRuntimeType` and many others.

The public API has been moving in the direction of named parameters, probably for good reason. The functions in this file by contrast are not exported and only used a few times, so the benefits of named parameters that leap to mind (any order, can save a set of arguments and pass them around) do not seem as straightforward.

But I defer to the group, thanks for the eyeballs.

Reply:
Actually, I think you're right, in this case we have a recursive function that is passing the exact same set of arguments save one, they should be combined.

Reply:
Hmmm, I'm looking at it, and it seems like similar refactoring could occur in so many places. I'm going to leave this as is for now.